app/models/pool_transaction_entry.rb

```diff
@@ -15,11 +15,42 @@ def income(address = nil)
   end
 
   def display_outputs(previews: false)
-    self.attributes["display_outputs"]
+    cell_inputs_for_display = cell_inputs.to_a.sort_by(&:id)
```

Comment:
cell_inputs is from where? I found no association with cell_inputs

Reply:
The `pool_transaction_entry` is deprecated, will not be used in the future.
We shall use `CkbTransaction.tx_pending` instead